test/integration/kpatch-test

```diff
@@ -152,7 +152,10 @@ build_module() {
 	log "build: $prefix"
 
 	if ! $KPATCHBUILD -n $modname $file >> $LOG 2>&1; then
-		[[ $shouldfail -eq 0 ]] && error "$prefix: build failed"
+		if [[ $shouldfail -eq 0 ]]; then
+			error "$prefix: build failed"
+			cp ${HOME}/.kpatch/build.log $(dirname ${LOG})/${prefix}.$(basename ${LOG})
```

Comment:
I am not familiar with the test framework code, I might be missing something. Should the copied over build failure log also removed during the invocation of the script again or the idea is to retain it for future reference.

Reply:
That's a good point. I think they should be treated the same as main log which [is removed on every start](https://github.com/dynup/kpatch/blob/master/test/integration/kpatch-test#L54). I'll post a v2, thank you.

Reply:
Similar to my other comments, this could just be `cp $HOME/.kpatch/build.log $prefix.log` which is more readable.